src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInMetricsListener.cs

```diff
@@ -63,7 +63,7 @@ public override void OnEventWritten(string name, object payload)
                 tags.Add(new KeyValuePair<string, object>(SemanticConventions.AttributeHttpFlavor, HttpTagHelper.GetFlavorTagValueFromProtocol(context.Request.Protocol)));
                 tags.Add(new KeyValuePair<string, object>(SemanticConventions.AttributeHttpScheme, context.Request.Scheme));
                 tags.Add(new KeyValuePair<string, object>(SemanticConventions.AttributeHttpMethod, context.Request.Method));
-                tags.Add(new KeyValuePair<string, object>(SemanticConventions.AttributeHttpStatusCode, context.Response.StatusCode.ToString()));
+                tags.Add(new KeyValuePair<string, object>(SemanticConventions.AttributeHttpStatusCode, context.Response.StatusCode));
```

Comment:
Use `(int)context.Response.StatusCode`? We have been explicitly type-casting it in other files.

Reply:
which file? we don't for ASP.NET Core as it is already `int`
https://github.com/open-telemetry/opentelemetry-dotnet/blob/c5ecd05c7220f012ca619305dd34ca0427605d4d/src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs#L247

Reply:
Ah I see. I got confused with the HttpInstrumentation files where the StatusCode is an enum. Never mind.